cms/djangoapps/contentstore/management/commands/import.py

```diff
@@ -22,14 +23,26 @@ def add_arguments(self, parser):
                             metavar='course_dir')
         parser.add_argument('--nostatic',
                             action='store_true',
-                            help='skip import of static content')
+                            help='Skip import of static content')
+        parser.add_argument('--nocodelib',
+                            action='store_true',
+                            help=(
+                                'Skip import of custom code library if it exists'
+                                '(NOTE: If static content is imported, the code library will also '
+                                'be imported and this flag will be ignored)'
+                            )),
```

Comment:
Yeah, I wasn't wild about this either. You're right about the logic with these flags. To put it another way: 

1. Static assets should be imported unless the client specifically requests otherwise
1. The code library should by imported, unless the client specifically requests otherwise, even if the client indicates that static assets should not be imported.

So if I pass in `--nostatic` without `--nocodelib`, the code library will indeed be imported. I think it makes sense behavior-wise to import the code library by default and force the client to switch it off. We may not even really need the flag to skip the code library import, but I included it to follow convention and to provide a sane path forward if a code library was exceptionally large and caused issues with the import. I agree that the logic is hard to follow. I can add some detail to the help text for both flags and add a log message in this command that indicates what is and is not being imported.